### PR TITLE
[Fix] fix call_to_consider_early_stop argument

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #111 , if available:*

*Description of changes:*
Change all call_to_consider_early_stop and call-to-consider-early-stop to early_stop_burnin_rounds and early-stop-burnin-rounds.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: training_scripts/gsgnn_np/arxiv_nc.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ gsf:
   input:
     restore_model_path: null
   output:
-    save_model_path: /data/ogbn-arxiv-1p/model/
-    save_embed_path: /data/ogbn-arxiv-1p/embed/
+    save_model_path: null
+    save_embed_path: null
```
Comment (thvasilo) [inline training_scripts/gsgnn_np/arxiv_nc.yaml:20]: Are these changes related to this PR? Or just a drive-by fix?

File: python/graphstorm/config/argument.py [hunk 2]
```diff
@@ -759,16 +759,16 @@ def no_validation(self):
 
     ### control early stop ###
     @property
-    def call_to_consider_early_stop(self):
-        """ Burning period calls to start considering early stop
+    def early_stop_burnin_rounds(self):
+        """ Burnin rounds before to start early stop
```
Comment (thvasilo) [inline python/graphstorm/config/argument.py:763]:
```suggestion
        """ Burn-in rounds before we start checking for the early stop condition.
```

**Review by thvasilo — APPROVED** [review]
Overall LGTM, might want to change the parameter description.

File: training_scripts/gsgnn_np/arxiv_nc.yaml (hunk 1 above)
Comment (zhjwy9343) [reply training_scripts/gsgnn_np/arxiv_nc.yaml:20]: It is a drive-by fix.

File: python/graphstorm/config/argument.py (hunk 2 above)
Comment (zhjwy9343) [reply python/graphstorm/config/argument.py:763]: Good. I will change other doc strings similar to this line.
